instrumentation/spring-webflux-5.0/spring-webflux-5.0.gradle

```diff
@@ -30,8 +30,7 @@ testSets {
 }
 
 dependencies {
-  compileOnly group: 'org.springframework', name: 'spring-webflux', version: '5.0.0.RELEASE'
-  compileOnly group: 'io.projectreactor.ipc', name: 'reactor-netty', version: '0.7.0.RELEASE'
+  implementation project(':instrumentation-core:spring:spring-webflux-5.0-core')
```

Comment:
check out #644 for how to shade this, so the agent and manual instrumentation don't conflict